genie/consortium_to_public.py

```diff
@@ -34,6 +39,131 @@ def commonVariantFilter(mafDf):
     return mafDf
 
 
+def _copyRecursive(
```

Comment:
nit: maybe this could be moved to one of the utility functions scripts like `genie/load.py` or `genie/process_function.py` for more centralized use of this function